test/neo4j_test.py

```diff
@@ -110,3 +181,16 @@ def test_construct_dwpc_query():
     dwpc = results['DWPC']
 
     assert dwpc == pytest.approx(0.03287590886921623)
+
+def test_create_path_return_clause():
```

Comment:
Let's discuss creating multiple tests. it is IDEAL to have a test test one thing and one thing only. There is possibility to paramtrize here.